### ISSUE TITLE
CodeBox Scaling Documentation

### ISSUE THREAD

**Issue by JamesFinlayson2** [shroominic/codeinterpreter-api#24]
It'd be great to get more information as to what's needed to scale this on codebox (and what, if anything, makes codebox a good place for this over any other environment). The 'team pro' accounts allow for unlimited private sandboxes - I assume it'd use that rather than 1 editor per session? What have you seen in terms of CodeBox limitations (I'm thinking of implementing this in an environment with 200+ users)?

**Comment by shroominic** [comment]
Yeah, I really need to write some documentation about this and set up some pricing plans. 
But we can also talk about this and create a custom deal. I am already running my discord bot (5000+ servers) on CodeBox and I am using AWS autoscaling so it should theoretically scale unlimited amounts as long as I pay for it haha. 
There will never be an unlimited plan because this is not manageable but there could be different subscriptions with fixed monthly usage and fixed price for any additional use. 
I am almost done with the backend I just need to finish some business logic then I can allow people in. 
So your 200+ users should be no problem when I got it working.

**Comment by shroominic** [comment]
Do you mean 200+ users are signed up for your app or 200+ users would actively use it at the same time?
Booth should be possible but still really different situations haha
